java/gradle.java/src/org/netbeans/modules/gradle/java/api/output/LocationOpener.java

```diff
@@ -80,7 +80,17 @@ public final void open() {
         }
     }
 
-    private int getMethodLine(final FileObject fo, final String methodName) {
+   private static String stripMethodParams(String methodNameWithParams) {
+        final int paramsIndex = methodNameWithParams.indexOf("(");
```

Comment:
Use indexOf('(') or add a //NOI18N at the end of line if you wish to keep it as a String.